model/jpa/src/main/java/org/keycloak/connections/jpa/DefaultJpaConnectionProviderFactory.java

```diff
@@ -143,17 +133,7 @@ public void init(Config.Scope config) {
     @Override
     public void postInit(KeycloakSessionFactory factory) {
         this.factory = factory;
-        checkJtaEnabled(factory);
-
-    }
-
-    protected void checkJtaEnabled(KeycloakSessionFactory factory) {
-        jtaLookup = (JtaTransactionManagerLookup) factory.getProviderFactory(JtaTransactionManagerLookup.class);
-        if (jtaLookup != null) {
-            if (jtaLookup.getTransactionManager() != null) {
-                jtaEnabled = true;
-            }
-        }
+        this.jtaLookup = (JtaTransactionManagerLookup) factory.getProviderFactory(JtaTransactionManagerLookup.class);
```

Comment:
@vramik @ahus1 I'm not sure about changes in this commit. It shouldn't be changed, right? As there can be non-JTA data sources provided, right?